### PR TITLE
[TizenFX] Update nuspec dependencies

### PR DESCRIPTION
**Pull request by JoonghyunCho**
### Description of Change ###
- This PR updates the dependencies information in nuspec file.
- `targetFramework ` values in `group`s will be used by Tizen runtime to set Tizen TFMs.
```
Tizen workload need to support `net6.0-tizen9.0` in advanced. 
```

### API Changes ###
 - N/A

### CODE REVIEW

File: pkg/Tizen.NET/Tizen.NET.nuspec [hunk 1]
```diff
@@ -40,19 +40,19 @@
...
         <dependency id="Tizen.NET.API10" version="10.0.0.17508" />
         <dependency id="Microsoft.NETCore.Platforms" version="3.1.0" />
       </group>
-      <group targetFramework=".NET6.0">
+      <group targetFramework="net6.0-tizen8.0">
         <dependency id="Tizen.NET.API11" version="11.0.0.18030" />
         <dependency id="Microsoft.NETCore.Platforms" version="6.0.9" />
       </group>
-      <group targetFramework=".NET6.0">
+      <group targetFramework="net6.0">
```
Comment (JongHeonChoi) [inline pkg/Tizen.NET/Tizen.NET.nuspec:51]:
```suggestion
      <group targetFramework="net6.0-tizen9.0">
        <dependency id="Tizen.NET.API12" version="$fxversion$" />
        <dependency id="Microsoft.NETCore.Platforms" version="6.0.9" />
      </group>
      <group targetFramework="net6.0">
```

**Comment by JongHeonChoi** [comment]
Isn't there a TFM called "net6.0-tizen"?
```<group targetFramework="net6.0-tizen">```
https://learn.microsoft.com/en-us/dotnet/standard/frameworks#os-version-in-tfms

**Review by JongHeonChoi — APPROVED** [review]

**Comment by JoonghyunCho** [comment]
> Isn't there a TFM called "net6.0-tizen"? `<group targetFramework="net6.0-tizen">` https://learn.microsoft.com/en-us/dotnet/standard/frameworks#os-version-in-tfms

The correct platform version needs to be specified when creating a package for a framework with a platform.

**Comment by JoonghyunCho** [comment]
I will ship it when the Tizen9.0 support for the workload is ready.

**Review by sung-su — APPROVED** [review]

**Review by shyunMin — APPROVED** [review]

**Comment by JongHeonChoi** [comment]
Please separate ```net6.0-tizen8.0``` and ```net6.0-tizen9.0```

**Pull request merged**